dvmkv2mp4

```diff
@@ -55,6 +55,7 @@ function cleanup {
   rm RPU.bin extra.json hdr10plus_metadata.json
   rm audio.exports
   rm sub.exports
+  rm *DV-MP4.asm *DV-MP4.*.srt *DV-MP4.*.sup
```

Comment:
asm file is something person may want i mean there wouldn't be asm file without -a switch

Reply:
Also those srt and sup are valid subtitle files. Some players can play Dolby vision with sup like emby on Nvidia shield.

Reply:
I see. What do you think about a new switch like -c | --cleanup to remove these files?

Reply:
more like no subs switch or something but it wouldhave have to respect -a switch. I mean if someone wants asm they use switch default it should not be created therefore it should not need to be cleaned up

Keep in mind the result MP4 does not have subs embedded as it sometimes failed and srt is widely supported so default was to keep subs alongside

So the only situation i see such switch is needed is if someone does not want to have subs at all. Then i wouldn't extract them at all and do processsubs and stuff instead of doing all that and then remove in cleanup :)